dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -227,10 +232,12 @@ EOD;
 	$bind_listenonv6 = $bind_listenonv6 == "" ? "none;" : $bind_listenonv6;
 	$bind_listenon = $bind_listenon == "" ? "none;" : $bind_listenon;
 	// Print "<pre>$bind_listenonv6 $bind_listenon";
+
+	$bind_listenport = $bind['listenport'] == "" ? "53" : $bind['listenport'];
 	if (array_key_exists("ipv6allow", $config['system'])) {
-		$bind_conf .= "\tlisten-on-v6 { $bind_listenonv6 };\n";
+		$bind_conf .= "\tlisten-on-v6 port $bind_listenport { $bind_listenonv6 };\n";
 	}
-	$bind_conf .= "\tlisten-on { $bind_listenon };\n";
+	$bind_conf .= "\tlisten-on port $bind_listenport { $bind_listenon };\n";
```

Comment:
Use braces around inline variable references (the new one and fix the older reference as well)